src/www/model/shadowsocks.ts

```diff
@@ -20,3 +20,8 @@ export interface ShadowsocksConfig {
   method?: string;
   name?: string;
 }
+
+// Endpoint to retrieve Shadowsocks proxy configurations.
+export interface ShadowsocksConfigSource {
```

Comment:
The ConfigSource doesn't need to live in the model. It can be defined in the repository module.